src/lib/side-bar/side-bar.component.ts

```diff
@@ -0,0 +1,41 @@
+import {Component, HostListener, Input, ViewEncapsulation} from '@angular/core';
+
+@Component({
+	moduleId:      module.id,
+	selector:      'hmn-side-bar',
+	templateUrl:   './side-bar.component.html',
+	styleUrls:     ['./side-bar.component.scss'],
+	host:          {class: 'hmn-sidebar__container'},
+	encapsulation: ViewEncapsulation.None
+})
+
+export class SideBarComponent
+{
+
+	/**
+	 * Main product color will be used as product title in logo
+	 *
+	 * @type {string}
+	 */
+	@Input('product-color')
+	public productColor: string;
+
+	/**
+	 * If is Collapsed is true Side bar will be collapsed
```

Comment:
`If is Collapsed is true Side bar will be collapsed`

I think that it should be:

`If true Side Bar will be collapsed`